extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/process/WithCategories.scala

```diff
@@ -2,9 +2,11 @@ package pl.touk.nussknacker.engine.api.process
 
 import pl.touk.nussknacker.engine.api.component.{ComponentId, SingleComponentConfig}
 
-// TODO: rename it? its no longer just a value with categories
+// TODO: This is deprecated API, remove it after ConfiCreator API will be removed
 case class WithCategories[+T](value: T, categories: Option[List[String]], componentConfig: SingleComponentConfig) {
 
+  def availableForCategory(category: String): Boolean = categories.isEmpty || categories.exists(_.contains(category))
```

Comment:
This can be removed - its implemented locally in one place that uses a method like this

Reply:
Good catch